riak_test/tests/legacy_s3_rewrite_test.erl

```diff
@@ -0,0 +1,95 @@
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2007-2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+-module(legacy_s3_rewrite_test).
+
+%% @doc `riak_test' module for testing object get behavior.
+
+-export([confirm/0]).
+-include_lib("erlcloud/include/erlcloud_aws.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-define(TEST_BUCKET, "legacy-s3-rewrite-test").
+
+confirm() ->
```

Comment:
Maybe it's about testing legacy code, so I'd say this is enough to test old behaviour.